src/root/components/OperationalUpdatePdfDocument/index.tsx

```diff
@@ -0,0 +1,874 @@
+  users: {
+    id: number;
+    first_name: string;
+    last_name: string;
+    email: string;
+    username: string;
+  }[];
+}
+
+interface Props {
+  operationalUpdateResponse: DrefOperationalUpdateApiFields;
+  drefOptions: DrefOptions;
+  // onLoad?: (params: { blob?: Blob }) => void;
+  strings: Strings;
+}
+
+function OperationalUpdatePdfDocument(props: Props) {
+  const {
+    operationalUpdateResponse,
+    drefOptions,
+    strings,
+  } = props;
+
+  const getMaps = () => {
+    if (!drefOptions) {
+      return [
+        {},
+        {},
+        {},
+      ];
+    }
+
+    return [
+      listToMap(drefOptions.planned_interventions, d => d.key, d => d.value),
+      listToMap(drefOptions.needs_identified, d => d.key, d => d.value),
+      listToMap(drefOptions.national_society_actions, d => d.key, d => d.value),
+    ];
+  };
+  const [
+    piMap,
+    niMap,
+    nsaMap,
+  ] = getMaps();
+
+  const getAffectedAreas = () => {
+    if (operationalUpdateResponse?.country_district) {
+      let areas = '';
+      const districts = operationalUpdateResponse.country_district.map(d => d.district_details);
+
+      districts.forEach(d => {
+        const names = d.map(dd => dd.name);
+        areas += names.join(', ');
+      });
+
+      return areas;
+    }
+
+    return undefined;
+  };
+  const affectedAreas = getAffectedAreas();
+
+  const logoUrl = '/assets/graphics/layout/go-logo-2020.png';
+
+  const isImminentOnset = operationalUpdateResponse?.disaster_type === ONSET_IMMINENT;
+  const documentTitle = [
+    operationalUpdateResponse?.country_district.map(d => d.country_details?.name).join(', '),
+    operationalUpdateResponse?.title
+  ].join(' | ');
+
+  const ifrcName: string = "International Federation of Red Cross and Red Crescent Societies (IFRC)";
+
+  return (
+    <Document
+      title={documentTitle}
+      author={ifrcName}
+      creator={ifrcName}
+      producer={ifrcName}
+      keywords="operationalUpdateResponse"
+    >
+      <PDFPage
+        style={pdfStyles.portraitPage}
+        size="A4"
+      >
+        <View style={pdfStyles.titleSection}>
+          <View style={pdfStyles.logoAndTitle}>
+            <PDFImage
+              style={pdfStyles.titleIfrcLogo}
+              src={logoUrl}
+            />
+            <Text style={pdfStyles.pageTitle}>
+              {strings.operationalUpdateExportTitle}
+            </Text>
+          </View>
+          <Text style={pdfStyles.subTitle}>
+            {[
+              operationalUpdateResponse?.country_district.map(d => d.country_details?.name).join(', '),
+              operationalUpdateResponse?.title
+            ].filter(Boolean).join(' | ')}
+          </Text>
+        </View>
+        {(operationalUpdateResponse.images_details?.map((el) => (
+          <View style={pdfStyles.section}>
+            <PDFImage
+              style={pdfStyles.bannerImage}
+              src={el?.file}
+            />
+          </View>
+        )))
```

Comment:
Use listVIew

Reply:
`ListView` is not applicable here since these are not the html tags (these are tags used by pdfjs)